src/DacpacTool/Extensions.cs

```diff
@@ -212,10 +251,22 @@ private static string EnsureIsDelimited(string name)
         {
             if (_ensureIsDelimitedMethod == null)
             {
-                _ensureIsDelimitedMethod = Type.GetType("Microsoft.Data.Tools.Schema.Common.FileUtils, Microsoft.Data.Tools.Utilities")
+                var method = Type.GetType("Microsoft.Data.Tools.Schema.Common.FileUtils, Microsoft.Data.Tools.Utilities");
```

Comment:
Perhaps rename this to variable to `fileUtilsType` as it is now no longer a method?